README.md

```diff
@@ -73,12 +73,10 @@ Usage: nix-installer install [OPTIONS] [PLAN]
        nix-installer install <COMMAND>
 
 Commands:
-  linux-multi
-          A standard Linux multi-user install
-  darwin-multi
-          A standard MacOS (Darwin) multi-user install
+  linux
```

Comment:
Should we also show the `macos` planner in this list?

Reply:
It's not in the linux binary, so I can't really show it without running the Mac binary which has no Linux entries.

Reply:
Fair enough,